R/mod_review_form_tbl.R

```diff
@@ -83,26 +83,26 @@ mod_review_form_tbl_server <- function(
       golem::cat_dev(form, "| Resetting userData\n")
       reload_data(reload_data() + 1)
       datatable_rendered(NULL)
-      session$userData$update_checkboxes[[form]] <- NULL
-      session$userData$review_records[[form]] <- data.frame(id = integer(), reviewed = character())
+      session$userData$pending_form_review_status[[form]] <- NULL
+      session$userData$pending_review_records[[form]] <- data.frame(id = integer(), reviewed = character())
     }) |> 
       bindEvent(active_subject(), form_review_data(), form_data())
     
     observeEvent(datatable_rendered(), {
       table_data(merged_form_data())
     }, ignoreInit = TRUE)
     
-    observeEvent(session$userData$update_checkboxes[[form]], {
+    observeEvent(session$userData$pending_form_review_status[[form]], {
       req(datatable_rendered())
       golem::cat_dev(form, "| Updating checkboxes\n")
       reload_data(reload_data() + 1)
-      checked <- session$userData$update_checkboxes[[form]]
+      checked <- session$userData$pending_form_review_status[[form]]
       df <- table_data() |> 
         dplyr::mutate(
-          o_reviewed = dplyr::if_else(
+          row_review_status = dplyr::if_else(
             subject_id == active_subject(), 
-            lapply(o_reviewed, modifyList, list(updated = checked)),
-            o_reviewed
+            lapply(row_review_status, modifyList, list(pending_form_review_status = checked)),
```

Comment:
```suggestion
            lapply(row_review_status, modifyList, list(updated = checked)),
```
Don't want to change this.

Reply:
Oops. Thanks for spotting this. It was somehow not captured in the unit tests. Let me have a look..